index.bs

```diff
@@ -2194,6 +2197,75 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.create Command ####  {#command-browsingContext-create}
+
+The <dfn export for=commands>browsingContext.create</dfn> command creates a new
+[=/browsing context=], either in a new window or in a new tab, and returns its
+[=browsing context id=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextCreateCommand = {
+        method: "browsingContext.create",
+        params: BrowsingContextCreateParameters
+      }
+
+      BrowsingContextCreateType = "tab" / "window"
+
+      BrowsingContextCreateParameters = {
+        type: BrowsingContextCreateType
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextCreateResult = {
+          context: BrowsingContext
+        }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.create">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let |type| be the value of the <code>type</code> field of
+     |command parameters|.
+
+  <!-- This is based on step 5 of https://w3c.github.io/webdriver/#new-window,
+       but without using the "current browsing context" concept. -->
+  1. Create a new [=top-level browsing context=] by running the [=window open
+     steps=] with <var ignore>url</var> set to "<code>about:blank</code>",
+     <var ignore>target</var> set to the empty string, and
+     <var ignore>features</var> set to "<code>noopener</code>". This must be
+     done without invoking the [=/focusing steps=] for the created browsing
+     context. If |type| is "<code>tab</code>", and the implementation supports
+     multiple browsing contexts in the same OS window, the new browsing context
+     should share an OS window with any other [=top-level browsing context=]. If
+     |type| is "<code>window</code>", and the implementation supports multiple
+     browsing contexts in separate OS windows, the created browsing context
+     should be in a new OS window. In all other cases the details of how the
+     browsing context is presented to the user are implementation defined.
+
+     Issue: HTML's [=window open steps=] depend on the "entry global object"
```

Comment:
We could have an optional context parameter that would be used as a reference? Like

```
1. Let |reference context id| be the value of the <code>context</code> field of parameters, if present, or null otherwise.
1. If |reference context id| is not null:
   1. Let |reference context| be the result of [=trying=] to [=get a browsing context=] with |reference context id|.
 1. Otherwise let |reference context| be null.
```

and then the "tab" case we say the new browsing context should share a window with the top-level browsing context of reference context, if it's not null, or with an implementation-defined existing context, if it is.

And then

Reply:
Yes, something like that sounds good! I'll do that in a follow-up PR.

Reply:
This is covered by https://github.com/w3c/webdriver-bidi/issues/151 now. Thanks @bwalderman for filing that.

Reply:
https://github.com/w3c/webdriver-bidi/pull/153